synapse/cortex.py

```diff
@@ -621,6 +625,69 @@ def onTagDel(self, name, func):
         # TODO allow name wild cards
         self.ontagdels[name].append(func)
 
+    def addRuntLiftHelp(self, form, func):
+        '''
+        Register a lift helper for a given form (and his sub props)
+        '''
+        self._runtLiftFuncs[form.full] = func
+        for name, prop in form.props.items():
```

Comment:
only register a fullly qualified property name here. do not do Form introspection.